runtime/common/src/crowdloan.rs

```diff
@@ -150,6 +150,9 @@ pub trait Config: frame_system::Config {
 		LeasePeriod=Self::BlockNumber,
 	>;
 
+	/// The type representing the leasing system.
+	type Leaser: Leaser<AccountId=Self::AccountId, LeasePeriod=Self::BlockNumber>;
```

Comment:
FYI `type Leaser` in `auctions.rs` has a unrelated doc.

Reply:
I fixed the doc in this PR?